.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: denoland/setup-deno@v1
```

Comment:
We can use [`denoland/setup-deno@v2`](https://github.com/denoland/setup-deno).

Reply:
I changed it.